scripts/build-images

```diff
@@ -16,9 +16,9 @@ xargs -n1 -t docker image pull --quiet << EOF >> build/images-core.txt
     ${REGISTRY}/rancher/hardened-coredns:v1.8.5-build20211119
     ${REGISTRY}/rancher/hardened-cluster-autoscaler:v1.8.5-build20211119
     ${REGISTRY}/rancher/hardened-dns-node-cache:1.21.2-build20211119
-    ${REGISTRY}/rancher/hardened-etcd:${ETCD_VERSION}-${IMAGE_BUILD_VERSION}
```

Comment:
if there's nothing left using `IMAGE_BUILD_VERSION` can you also remove it from version.sh to avoid confusion?

Reply:
It's not being used. Done in https://github.com/rancher/rke2/pull/2171/commits/68303a64afce935f5c5f0dd387c7c22d341587a1